src/types.ts

```diff
@@ -57,41 +60,28 @@ export interface JudgeReviews {
   questions: [Questions];
   final: number;
 }
-export const categories = [
-  "hotel dieu shaver health and rehabilitation",
-  "ite challenge",
-  "beasley neighbourhood association challenge",
-  "best overall hack",
-  "best education hack",
-  "best finance hack",
-  "best environment hack",
-  "best health hack",
-  "best quality of life/productivity hack",
-  "innovation factory challenge",
-  "td challenge",
-  "arcelormittal dofasco challenge",
-  "materials challenge",
-  "algorand challenge",
-  "hypercare challenge",
-  "best uipath automation hack",
-  "best domain registered with domain.com",
-  "best use of google cloud",
-  "best use of mongodb atlas",
-  "best use of blockstack",
-  "most creative radar.io hack"
-].map(each => {
-  return each
-    .split(" ")
-    .map(word => {
-      return (
-        word.substring(0, 1).toUpperCase() +
-        word.substring(1, word.length).toLowerCase()
-      );
-    })
-    .join(" ");
-});
+
+// .map(each => {
```

Comment:
Can prob just remove this comment block since the logic is actually incorporated somewhere else now